### ISSUE TITLE
save_config, write mem timeout issue still on Netmiko 4.1.1

### ISSUE THREAD

**Issue by onlywifi** [ktbyers/netmiko#2843]
I'm still having this issue with Netmiko 4.1.1 when saving a long config that takes around 25 seconds to finsish.
I've added the 'fast_cli'=False as both argument and global command but it is not fixing it:
This is my current code:
```
wlc1 = {"device_type":"cisco_ios","ip":"wlc1","username":"admin","password":"admin","fast_cli":"False"}
wlc2 = {"device_type":"cisco_ios","ip":"wlc2","username":"admin","password":"admin","fast_cli":"False"}
wlcList = [wlc1,wlc2]
!
!
net_connect.fast_cli = False			#line 196
output += net_connect.save_config()		#line 197
net_connect.fast_cli = True 			#line 198
```

And the error:
```
  File "list_v10a.py", line 197, in <module>
    output += net_connect.save_config()
  File "c:\Python39\lib\site-packages\netmiko\cisco\cisco_ios.py", line 58, in save_config
    return super().save_config(
  File "c:\Python39\lib\site-packages\netmiko\cisco_base_connection.py", line 252, in save_config
    output = self._send_command_str(
  File "c:\Python39\lib\site-packages\netmiko\base_connection.py", line 1716, in _send_command_str
    output = self.send_command(*args, **kwargs)
  File "c:\Python39\lib\site-packages\netmiko\utilities.py", line 596, in wrapper_decorator
    return func(self, *args, **kwargs)
  File "c:\Python39\lib\site-packages\netmiko\base_connection.py", line 1694, in send_command
    raise ReadTimeout(msg)
netmiko.exceptions.ReadTimeout:
Pattern not detected: 'WLC1\\#' in output.

Things you might try to fix this:
1. Explicitly set your pattern using the expect_string argument.
2. Increase the read_timeout to a larger value.
```

**Comment by ktbyers** [comment]
@onlywifi Can you do a `python3 -m pip list` and post the output here:

The above error does not look like Netmiko 4.1.1. In other words, line 252 is not this (in Netmiko 4.1.1):

```
  File "c:\Python39\lib\site-packages\netmiko\cisco_base_connection.py", line 252, in save_config
    output = self._send_command_str(
```

**Comment by ktbyers** [comment]
Closing as it looks like an environment issue (i.e. not actually running Netmiko 4.1.1).

Just re-open this issue if you are still working on it.

**Comment by onlywifi** [comment]
Hi Kirk and so sorry for my late response as I have been off for the last 3 weeks.

This is the output:
:\Python39\python.exe: No module named piplist

C:\Python39>python.exe -m pip list
Package        Version
-------------- ---------
asgiref        3.5.2
bcrypt         3.2.2
cffi           1.15.0
colorama       0.4.4
cryptography   37.0.2
DateTime       4.3
distlib        0.3.4
Django         4.0.5
filelock       3.7.1
future         0.18.2
image          1.5.33
MouseInfo      0.1.3
netmiko        4.1.0
ntc-templates  3.0.0
paramiko       2.11.0
Pillow         9.1.1
pip            22.1.2
platformdirs   2.5.2
PyAutoGUI      0.9.53
pycparser      2.21
PyGetWindow    0.0.9
PyMsgBox       1.0.9
PyNaCl         1.5.0
pyperclip      1.8.2
PyRect         0.2.0
PyScreeze      0.1.28
pyserial       3.5
pytweening     1.0.4
pytz           2021.3
PyYAML         6.0
qrcode         7.3.1
scp            0.14.4
setuptools     62.6.0
six            1.16.0
sqlparse       0.4.2
tenacity       8.0.1
textfsm        1.1.2
timedelta      2020.12.3
tzdata         2022.1
virtualenv     20.14.1
wheel          0.37.1
zope.interface 5.4.0

[notice] A new release of pip available: 22.1.2 -> 22.2
[notice] To update, run: python.exe -m pip install --upgrade pip
